app/forgot-password/page.tsx

```
@@ -0,0 +1,80 @@
+    } else {
+      setSubmitted(true);
+    }
+  };
+
+  if (submitted) {
+    return (
+      <div className="flex min-h-screen items-center justify-center">
+        <div className="text-center space-y-4">
+          <h1 className="text-2xl font-bold">Check your email</h1>
+          <p className="text-muted-foreground">
+            If that email exists, we've sent a password reset link.
```

ESLint

[error] `'` can be escaped with `&apos;`, `&lsquo;`, `&`#39`;`, `&rsquo;`.

(react/no-unescaped-entities)